modules/georepl-prereqs.adoc

```diff
@@ -1,7 +1,14 @@
 [[georepl-prereqs]]
 = Geo-replication requirements and constraints
 
-* A single database, and therefore all metadata and Quay configuration, is shared across all regions.
+[IMPORTANT]
+====
+* In geo-replicated setups, {productname} is unable to withstand loss of storage. For example, if a customer loses storage in one geographic region, it is not saved in another geo-replicated environment. Geo-replication requires that all regions are able to read/write to all other region's storage. Storage must be geographically accessible by all other regions. Inter-regional storage failures are not offered graceful exits by {productname}. 
```

Comment:
```suggestion
* In geo-replicated setups, {productname} requires that all regions are able to read/write to all other region's object storage. Object storage must be geographically accessible by all other regions. 
* {productname} has no internal awareness of the health or availability of the connected object storage system. If the object storage system of one site becomes unavailable there will be no automatic redirect to the remaining storage system(s) of the remaining site(s)
* In case of a object storage system failure of one geo-replicating site, that site's Quay deployment needs to be shutdown as well so that clients get redirect to the remaining site with intact storage systems by a global load balancer otherwise clients will experience pull and push failures
* Geo-replication is asynchronous by nature. The permanent loss of site will incur loss of the data that has been saved in that sites' object storage system but has not yet been replicated to the remaining sites at the time of failure.
```